### PR TITLE
Fix dmg building issues

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

**Comment by Yeti-or** [comment]
oh shit, need to rebase and fix paths

**Comment by awinogradov** [comment]
@Yeti-or you can) I know)

**Comment by Yeti-or** [comment]
WIP

File: build/webpack.production.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    output: Object.assign({}, base.output, {
+        publicPath: '../../dist'
+    }),
+
+    plugins: [
+        new webpack.optimize.OccurenceOrderPlugin(),
+        new webpack.DefinePlugin({
+            '__DEV__': true,
+            'process.env': {
+                'NODE_ENV': JSON.stringify('development')
```
Comment (awinogradov) [inline build/webpack.production.js]: production

File: package.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  packager(opts, cb);
+}
+
+
+function log(plat, arch) {
+  return (err, filepath) => {
+    if (err) return console.error(err);
+    console.log(`${plat}-${arch} finished!`);
+  };
+}
```
Comment (awinogradov) [inline package.js]: 4 spaces

File: build/webpack.production.js (hunk 1 above)
Comment (Yeti-or) [inline build/webpack.production.js]: да  ага

File: package.js (hunk 2 above)
Comment (Yeti-or) [inline package.js]: не смотри пока на pack
он не готов

**Comment by Yeti-or** [comment]
I give up on this. Let's move to https://github.com/theprotein/protein-dynamics-revised

**Pull request closed without merging**